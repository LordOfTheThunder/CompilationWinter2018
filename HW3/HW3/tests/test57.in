void TestSpace() {
	byte b = 100 b;
}
