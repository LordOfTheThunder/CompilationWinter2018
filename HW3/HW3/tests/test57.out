---end scope---
b BYTE 0
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
TestSpace ()->VOID 3
